plugins/vendor-example/meson.build.in

```diff
@@ -0,0 +1,23 @@
+{%- if Parent in ['Usb', 'Hid'] -%}
+if gusb.found()
```

Comment:
For hid need to make sure it's linux, and don't think you need gusb for sure do you?

Reply:
Ohh this is FuHidDevice which is a superclass of FuUsbDevice -- it's not the hidraw stuff -- that is FuUdev as it uses ioctls.